dataprofiler/profilers/utils.py

```diff
@@ -87,8 +88,9 @@ def shuffle_in_chunks(data_length, chunk_size):
     if not data_length or data_length == 0 \
        or not chunk_size or chunk_size == 0:
         return []
-    
-    rng = np.random.default_rng()
+
+    rng = np.random.default_rng(int(settings._seed))
+
     if 'DATAPROFILER_SEED' in os.environ:
```

Comment:
for this if statement, we should only override with env var if `dp._seed is None` e.g. if a user calls `dp.set_seed` the env var shouldn't override it.

Reply:
`if 'DATAPROFILER_SEED' in os.environ and settings._seed is None:`

Reply:
Okay, will fix